src/open_inwoner/openzaak/admin.py

```diff
@@ -57,8 +59,33 @@ def queryset(self, request, queryset):
         return queryset
 
 
+class ZaakTypeInformatieObjectTypeConfigInline(admin.TabularInline):
+    model = ZaakTypeInformatieObjectTypeConfig
+    fields = [
+        "omschrijving",
+        "document_upload_enabled",
+        "informatieobjecttype_url",
+        "zaaktype_uuids",
+    ]
+    readonly_fields = [
+        "omschrijving",
+        "informatieobjecttype_url",
+        "zaaktype_uuids",
+    ]
+    ordering = ("omschrijving",)
+
+    def has_add_permission(self, request, obj):
+        return False
+
+    def has_delete_permission(self, request, obj=None):
```

Comment:
I am not sure maybe I didn't understand something but do we want to allow delete? I mean since we just show data that we retrieve and the next day it will be again there. This has to do generally with the delete operation (ZaakTypeConfig as well).

Reply:
Allowing delete like this is mostly for development and testing purposes, which is why I put it as superuser only.